bee-network/bee-autopeering/src/discovery/manager.rs

```diff
@@ -551,13 +550,14 @@ fn validate_verification_response(
 ) -> Result<RequestValue, ValidationError> {
     use ValidationError::*;
 
-    if let Some(reqv) = request_mngr.write().remove::<VerificationRequest>(peer_id) {
+    if let Some(reqv) = request_mngr.remove_request::<VerificationRequest>(peer_id) {
         if verif_res.request_hash() == reqv.request_hash {
             let res_services = verif_res.services();
             if let Some(autopeering_svc) = res_services.get(AUTOPEERING_SERVICE_NAME) {
                 if autopeering_svc.port() == source_socket_addr.port() {
                     Ok(reqv)
                 } else {
+                    // Fix: Update the outdated/wrong port
```

Comment:
Is it fixed already? If so, we could remove the comment.

Reply:
yup